src/SarifTests/SarifUtilitiesTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.CodeAnalysis.Sarif;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Sarif.Tests
+{
+    [TestClass()]
+    public class SarifUtilitiesTests
+    {
+        private const string V1_0_0 = "1.0.0";
+        private const string V1_0_0_BETA_5 = "1.0.0-beta.5";
+
+        [TestMethod()]
+        public void ConvertToSchemaUriTest1()
+        {
+            V1_0_0.ConvertToSarifVersion().ConvertToSchemaUri();
```

Comment:
A unit test has three parts:

1. Prepare to perform some action (optional: in this case, there's no preparation to do).
2. Perform the action.
3. Verify that the action had the correct result.

So you've done #2 but not #3.

You need to _assert_ that the action you've taken had the correct result. For now, let's use the assertion library that comes with Visual Studio (I actually prefer another library, but this will get us started quickly):

```
Uri uri = SarifVersion.OneZeroZero.ConvertToSchemaUri();
Assert.AreEqual(uri.ToString(), "http://json.schemastore.org/sarif-1.0.0")
```